roles/confluent.kafka_rest/defaults/main.yml

```diff
@@ -19,9 +19,11 @@ kafka_rest_final_java_args: "{{ kafka_rest_java_args + [ kafka_rest_custom_java_
 kafka_rest_service_overrides:
   User: "{{ kafka_rest_user if kafka_rest_user != kafka_rest_default_user else '' }}"
   Group: "{{ kafka_rest_group if kafka_rest_group != kafka_rest_default_group else '' }}"
+  ExecStart: "{% if installation_method == 'archive' %}{{ kafka_rest.server_start_file }} {{ kafka_rest.config_file }}{% endif %}"
 
 kafka_rest_service_environment_overrides:
-  LOG_DIR: /var/log/kafka-rest
+  #LOG_DIR: /var/log/kafka-rest
```

Comment:
remove commented code

Reply:
Ok. I kept the comment there because that directory is in conflict with the default directory.
Repo install -> LOG_DIR=/var/log/confluent/kafka-rest
cp-ansible "package" deployment -> LOG_DIR=/var/log/kafka-rest (hard-coded in override)
cp-ansible "archive" deployment -> LOG_DIR=/var/log/rest-proxy (defined by {{kafka_rest.appender_log_path}}, propagated to override)

I will gladly remove the comment, but I was worried it would be overlooked.